shared/protocol/src/main/java/io/pravega/shared/protocol/netty/FailingRequestProcessor.java

```diff
@@ -54,7 +54,12 @@ public void append(Append appendData) {
     public void readSegment(ReadSegment readSegment) {
         throw new IllegalStateException("Unexpected operation");
     }
-    
+
+    @Override
+    public void flushToStorage(WireCommands.FlushToStorage flushToStorage) {
```

Comment:
I think that, in general, the `WireCommand` naming pattern is like:
- Request: DO_SOMETHING
- Reply: SOMETHING_DONE
In this case, we could follow this like:
- Request: `FlushStorage`
- Reply: `StorageFlushed`.

Reply:
Changed!